test/testutil.js

```diff
@@ -68,14 +68,20 @@ class TestHarness {
     return this.user;
   }
 
-  async setupStitchClient() {
+  async setupUserAuthAndStitchClient() {
     await this.configureUserpass();
     await this.createUser();
 
     this.stitchClient = new stitch.StitchClient(this.testApp.client_app_id, { baseUrl: this.serverUrl });
     await this.stitchClient.authenticate('userpass', this.userCredentials);
   }
 
+  async setupStitchClient() {
```

Comment:
[minor] These two methods can be combined with a parameter for `shouldConfigureAuth`